examples/text_classification/iterable_train.py

```diff
@@ -25,7 +49,22 @@ def generate_batch(batch):
     return text, offsets, label
 
 
+r"""
+torch.utils.data.DataLoader is recommended for PyTorch users to load data.
+We use DataLoader here to load datasets and send it to the train()
+and text() functions.
+"""
+
+
 def train(lr_, num_epoch, data_):
+    r"""
+    Here we use SGD optimizer to train the model.
```

Comment:
We still use different optimization approaches across the tutorial, train.py and iterable_train.py. Could you look into unifying them?